src/main/java/org/fundacion/jala/converter/view/metadata/MetaDataInterface.java

```diff
@@ -99,6 +101,7 @@ public void actionPerformed(final ActionEvent e) {
 
     /**
      * Obtains the request
+     *
      * @throws IOException
```

Comment:
Description of the exception is missing

Reply:
Is missing a description of exception

Reply:
then is missing "."  a the end of the javafoc